stdlib/tkinter/ttk.pyi

```diff
@@ -1058,7 +1064,7 @@ class Treeview(Widget, tkinter.XView, tkinter.YView):
     def identify_column(self, x: int) -> str: ...
     def identify_region(self, x: int, y: int) -> Literal["heading", "separator", "tree", "cell", "nothing"]: ...
     def identify_element(self, x: int, y: int) -> str: ...  # don't know what possible return values are
-    def index(self, item: str) -> int: ...
+    def index(self, item: str | int) -> int: ...
     def insert(
         self,
         parent: str,
```

Comment:
Just below this line (can't leave a comment there unfortunately) there is `id: str = ...` and `iid: str | None = None`. Those should also accept ints.